api/specs.proto

```diff
@@ -258,6 +258,12 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	message SecurityConfig {
+	    string userns = 30;
+	    string credential_spec = 31 [(gogoproto.customname) = "CredentialSpec"];
```

Comment:
Basically because...

![dog](https://cloud.githubusercontent.com/assets/260642/22811733/9e79f818-ef0e-11e6-9811-fbd95d7e191a.jpg)

In all seriousness, I saw that `registry_auth` was high so I put this one to start at 30.  Completely arbitrary and I defer to you for what it should be :)